manifests/openshift-templates/list.yaml

```diff
@@ -20,10 +20,26 @@ objects:
     version: v1alpha1
     scope: Namespaced
     names:
-      plural: meterings
-      singular: metering
-      kind: Metering
-      listKind: MeteringList
+      plural: meteringconfigs
+      singular: meteringconfig
+      kind: MeteringConfig
+      listKind: MeteringConfigList
+
+- apiVersion: apiextensions.k8s.io/v1beta1
+  kind: CustomResourceDefinition
+  metadata:
+    name: hivetables.metering.openshift.io
+    annotations:
+      catalog.app.coreos.com/displayName: "Metering Presto Table"
+      catalog.app.coreos.com/description: "A table within PrestoDB"
+  spec:
+    group: metering.openshift.io
+    version: v1alpha1
```

Comment:
do we need the v1 versions too?

Reply:
We do, when I switched the manifests over, v1 is included